extensions/wikia/FeedsAndPosts/scripts/index.js

```diff
@@ -8,12 +8,13 @@ require(['jquery'], function ($) {
 			var wikiName = $('meta[property="og:site_name"]').prop('content');
 
 			// Load FP into a newly inserted element
-			$('#wikia-recent-activity').after('<div class="rail-module feed-posts-module"></div>');
+			$('#recirculation-rail').append('<div class="rail-module feed-posts-module"></div>');
 			var fpContainer = $('.feed-posts-module').get(0);
 			window.fp.default(fpContainer, {
```

Comment:
not related, but considering IDs as globals I'd say `fp` can be overridden via some id in custom markup much easier that sth like `feedsAndPosts`

Reply:
I'd rather go with something like `nbgToAQTol` :D